tests/loader/test_tanks_and_temples_loader.py

```diff
@@ -0,0 +1,201 @@
+"""Unit tests for Tanks & Temple dataset loader.
+
+Author: John Lambert
+"""
+
+import unittest
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import numpy as np
+import open3d
+from gtsam import Cal3Bundler, Unit3
+
+import gtsfm.utils.geometry_comparisons as geom_comp_utils
+import gtsfm.visualization.open3d_vis_utils as open3d_vis_utils
+from gtsfm.common.image import Image
+from gtsfm.loader.tanks_and_temples_loader import TanksAndTemplesLoader
+from gtsfm.frontend.verifier.loransac import LoRansac
+
+
+_TEST_DATA_ROOT = Path(__file__).resolve().parent.parent / "data" / "tanks_and_temples_barn"
+
+
+class TanksAndTemplesLoaderTest(unittest.TestCase):
+    def setUp(self) -> None:
+        scene_name = "Barn"  # 'Truck'
+
+        # Uncomment lines below to run locally.
+        # _TEST_DATA_ROOT = Path('/Users/johnlambert/Downloads/Tanks_and_Temples_Barn_410')
+        # lidar_ply_fpath = _TEST_DATA_ROOT / f'{scene_name}.ply'
+        # colmap_ply_fpath = _TEST_DATA_ROOT / f'{scene_name}_COLMAP.ply'
```

Comment:
Why is there a difference between how it works locally and on the CI? Is this done in any other tests?

Reply:
Good point, removed now. This really was more of an "integration test", so I moved it to a visualization script for Tanks and Temples (`gtsfm/visualization/visualize_scene_tanks_and_temples.py`).